src/styles/variables.less

```diff
@@ -17,3 +19,12 @@
 @navbar-default-link-hover-color: @white;
 
 @section-min-height: 250px;
+
+@big-letter-spacing: 5px;
```

Comment:
Hai boss nu mai muta variabilele in `variables.less`. Te bat! :smile: 

Să-mi spui mie unde ai mai refolosi asta în afară de `main.less` dacă crezi că locul ei e aici.